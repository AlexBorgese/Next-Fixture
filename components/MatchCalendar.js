import React from 'react';
import { View, Text } from 'react-native';

export default class MatchCalendar extends React.Component {
	render() {
		console.log('calendar');
		return (
			<View>
				<Text>Next Fixture</Text>
			</View>
		);
	}
}

MatchCalendar.displayName = 'MatchCalendar';
