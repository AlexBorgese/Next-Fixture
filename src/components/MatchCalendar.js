import React from 'react';
import { View } from 'react-native';
import { Calendar } from 'react-native-calendars';

const MatchCalendar = () => {
	return (
		<View>
			<Calendar />
		</View>
	);
};

MatchCalendar.displayName = 'MatchCalendar';

export default MatchCalendar;
