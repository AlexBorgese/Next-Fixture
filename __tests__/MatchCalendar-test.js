import React from 'react';
import { shallow } from 'enzyme';

import MatchCalendar from '../src/components/MatchCalendar';

describe('Given the MatchCalendar component', () => {
	let component;
	describe('when rendered', () => {
		beforeAll(() => component = shallow(<MatchCalendar />));

		it('should render the correct snapshot', () => {
			expect(component).toMatchSnapshot();
		});
	});
});
