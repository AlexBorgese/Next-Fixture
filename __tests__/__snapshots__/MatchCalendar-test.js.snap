// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the MatchCalendar component when rendered should render the correct snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <View>
    <Calendar />
</View>,
  "nodes": Array [
    <View>
      <Calendar />
</View>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <MatchCalendar />,
    "_instance": null,
    "_newState": null,
    "_rendered": <View>
      <Calendar />
</View>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <MatchCalendar />,
}
`;
